dgp/utils/render_3d_to_2d.py

```diff
@@ -0,0 +1,420 @@
+# Copyright 2023 Toyota Motor Corporation.  All rights reserved.
+import json
+import logging
+import math
+import os
+import time
+from collections import defaultdict
+from multiprocessing import Pool
+from typing import Optional, Tuple
+
+import numpy as np
+from matplotlib import pyplot as plt
+
+from dgp.annotations import ANNOTATION_TYPE_TO_ANNOTATION_GROUP
+from dgp.constants import ALL_ANNOTATION_TYPES, DATASET_SPLIT_NAME_TO_KEY
+from dgp.datasets.synchronized_dataset import (
+    SynchronizedScene,
+    SynchronizedSceneDataset,
+)
+from dgp.proto.dataset_pb2 import SceneDataset
+from dgp.utils.camera import Camera
+from dgp.utils.structures.bounding_box_3d import BoundingBox3D
+
+ANNOTATIONS_3D = 'bounding_box_3d'
+ANNOTATIONS_2D = 'bounding_box_2d'
+# The list of cameras' name
+FRONT_CAMERA = ["CAMERA_21"]
```

Comment:
If this parameter is not used in source code, I think it's better to remove it to avoid confusion

Reply:
@yuta-tsuzuki-woven This is needed to run the unit test.

Reply:
Is it possible to move this variable to the test code?

Reply:
Thank you, Done.